.github/workflows/ghci.yml

```diff
@@ -46,7 +63,11 @@ jobs:
 
       - name: install deps
         run: |
-          case "${{ matrix.os }}" in
+          pkgflavour="${{ matrix.docker_image }}"
+          if [ -z "$pkgflavour" ] ; then
+            pkgflavour="${{ matrix.os }}"
+          fi
+          case "$pkgflavour" in
```

Comment:
In `jobs.build.name`, you use `${{ matrix.docker_image || matrix.os }}`. Can’t that be used here as well?

Reply:
Yup, it can and now does, thx.

Reply:
Just a note: you can now inline `pkgflavour` if you’d like.

Reply:
Done, thanks.